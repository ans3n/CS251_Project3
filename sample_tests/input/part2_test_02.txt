string int
insert Enz2V8 786
size
empty
insert 077IP -98
peek 077IP
print
insert lwHROdo 48
print
extract lwHROdo
peek 077IP
maximum_key
print
insert xStZAym 501
minimum_key
insert in3IvHP -106
insert vbRRlY63 -756
insert Q93vEiG 829
insert FgJlUIe -199
size
peek vbRRlY63
maximum_key
extract FgJlUIe
peek xStZAym
extract xStZAym
minimum_key
insert uCLq 110
extract uCLq
peek in3IvHP
extract Q93vEiG
quit
