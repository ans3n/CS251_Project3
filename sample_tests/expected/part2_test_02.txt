insert Enz2V8 786
size
1
empty
false
insert 077IP -98
peek 077IP
-98
print
077IP -> -98
└R: Enz2V8 -> 786
insert lwHROdo 48
print
lwHROdo -> 48
└L: Enz2V8 -> 786
 └L: 077IP -> -98
extract lwHROdo
48
peek 077IP
-98
maximum_key
Enz2V8
print
Enz2V8 -> 786
└L: 077IP -> -98
insert xStZAym 501
minimum_key
077IP
insert in3IvHP -106
insert vbRRlY63 -756
insert Q93vEiG 829
insert FgJlUIe -199
size
7
peek vbRRlY63
-756
maximum_key
xStZAym
extract FgJlUIe
-199
peek xStZAym
501
extract xStZAym
501
minimum_key
077IP
insert uCLq 110
extract uCLq
110
peek in3IvHP
-106
extract Q93vEiG
829
quit
